DESCRIPTION

```diff
@@ -1,14 +1,15 @@
 Package: IsoMemo
-Title: Package to Retrieve Data from the Max Plank Institute IsoMemo API
-Version: 1.0.5
-Date: 2022-04-18
+Title: Retrieve Data using the Max Plank Institute 'IsoMemo' API 
+Version: 1.0.5.001
+Date: 2022-05-03
 Authors@R: c(
             person("Jianyin", "Roachell", ,"jianyin.roachell@inwt-statistics.de","cre"),
             person("Andreas", "Neudecker", , "andreas.neudecker@inwt-statistics.de","aut"),
             person("Ricardo", "Fernandes", , "fernandes@shh.mpg.de","aut")
             )
-Description: API wrapper that contains functions to retrieve data from the IsoMemo partnership databases and perform analysis given three parameters - database, category, and field.
+Description: API wrapper that contains functions to retrieve data from the 'IsoMemo' partnership databases. Webservices for API: {https://isomemodb.com/api/v1/}. 
```

Comment:
Not curly, but angle brackets. From the [r docs](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#The-DESCRIPTION-file):

> The mandatory ‘Description’ field should give a comprehensive description of what the package does. One can use several (complete) sentences, but only one paragraph. It should be intelligible to all the intended readership (e.g. for a CRAN package to all CRAN users). It is good practice not to start with the package name, ‘This package’ or similar. As with the ‘Title’ field, double quotes should be used for quotations (including titles of books and articles), and single quotes for non-English usage, including names of other packages and external software. This field should also be used for explaining the package name if necessary. URLs should be enclosed in angle brackets, e.g. ‘<https://www.r-project.org>’: see also [Specifying URLs](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Specifying-URLs).

Reply:
yes, i will fix with <> brackets